docs/reference/units/template.rst

```diff
@@ -48,6 +48,14 @@ Template-Specific Fields
     translation into other languages. It must be one line long, ideally it
     should be short (50-70 characters max).
 
+    Example: if a template has a ``summary`` field is set to:
+
+        *Test serial connection for port {serial_port}*
+
```

Comment:
```suggestion

Each instantiated job will include the same ``summary`` field, generated with the actual value of the serial port.

```

Reply:
As discussed elsewhere, this is a good idea to explain what happens to these generated jobs, but it might be better to put it in the examples sections at the bottom of the document. I will do that next.